Makefile

```diff
@@ -1,6 +1,10 @@
 release:
+ifndef VERSION
+	$(error VERSION is undefined)
+endif
 	git describe --exact
```

Comment:
As suggested by the line, this might be the last Python repository where the release process requires to ceate a tag before uploading to PyPI. If desired, I could probably work on a PR switching to the version file behavior as for the downstream `scc` and `omego` packages

Reply:
No objections, but the `ifndef VERSION` strategy would likely remain. I use this same Makefile for `omego` already.